javascript-modal/main.js

```diff
@@ -0,0 +1,27 @@
+var $modalButton = document.querySelector('.modal-button');
+$modalButton.addEventListener('click', modalVisibilityOn);
+
+var $modelElement = document.querySelector('.modal');
+
+var $noButton = document.querySelector('.no-button');
+$noButton.addEventListener('click', modalVisibilityOff);
+
+var $overlayBG = document.querySelector('#overlay');
+
+var visibilitOff = true;
+
+function modalVisibilityOn(event) {
+  if (visibilitOff) {
```

Comment:
This conditional isn't necessary. You applied this directly to the button to make the modal visible, so you know exactly what you want to do every time it is clicked: show the modal.